benchmarks/static_runtime/deep_wide_pt_bench.cc

```diff
@@ -99,6 +114,7 @@ static void BM_deep_wide_static_threaded(benchmark::State& state) {
 }
 
 BENCHMARK(BM_deep_wide_base)->RangeMultiplier(8)->Ranges({{1, 20}});
+BENCHMARK(BM_deep_wide_fast)->RangeMultiplier(8)->Ranges({{1, 20}});
```

Comment:
while you're on it - you can make this  and the ones below BENCHMARK_RELATIVE then the comparison will look much nicer

Reply:
I think that's a folly feature, but AFAICT we're not using folly here.